index.html

```diff
@@ -68,8 +72,9 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
             </h3>
 
             <div id="playerControls">
-                <button class="btn" id="togglePlayer">
+                <button class="mybtn" id="togglePlayer">
```

Comment:
btn is a predefined class of bootstrap. I wished to overwrite some of its properties. Using !important was not a great option, so designed the button afresh